### DISCUSSION TITLE
Creating Sub Teams

### DISCUSSION THREAD

**Discussion by JamesK254** [KenyaFlutterDev/flutter_ke#16] [General]
Hello everyone I think we should create sub teams each assigned a role to do, like:
1. MobileApp Backend Team
2. Webapp Backend Team
3. Mobile Frontend
4. Web Fronted

And pre-define on what each sub team is supposed to build.

**Comment by mboyamike** [comment]
Umm... first the backend would be similar (shared between mobile and web)

At the moment we don't have specific teams. We instead accept contributions from any interested party. If the interest grows much bigger we'll organise the specific teams

What's your area of expertise?
